analytic_engine/src/sst/meta_data/cache.rs

```diff
@@ -62,7 +70,7 @@ impl MetaData {
                 file_meta_data.num_rows(),
                 file_meta_data.created_by().map(|v| v.to_string()),
                 // Remove the key value metadata.
-                None,
+                Some(other_kv_metas),
```

Comment:
The comment can be changed.
Here if `other_kv_metas` is empty, do we need to pass a `None`?